docker/dev/.env.example

```diff
@@ -9,6 +9,18 @@ GATEHUB_GATEWAY_UUID=
 GATEHUB_VAULT_UUID_EUR=
 GATEHUB_VAULT_UUID_USD=
 GATEHUB_SETTLEMENT_WALLET_ADDRESS=
+SEND_EMAIL_RATE_LIMIT=1
+SEND_EMAIL_RATE_LIMIT_PAUSE_IN_SECONDS=3600
+SEND_EMAIL_RATE_LIMIT_RESET_INTERVAL_IN_SECONDS=3600
+LOGIN_RATE_LIMIT=3
+LOGIN_RATE_LIMIT_PAUSE_IN_SECONDS=600
+LOGIN_RATE_LIMIT_RESET_INTERVAL_IN_SECONDS=3600
+LOGIN_IP_RATE_LIMIT=30
+LOGIN_IP_RATE_LIMIT_PAUSE_IN_SECONDS=3600
+LOGIN_IP_RATE_LIMIT_RESET_INTERVAL_IN_SECONDS=3600
+LOGIN_IP_BLOCK_RATE_LIMIT=500
+LOGIN_IP_BLOCK_RATE_LIMIT_PAUSE_IN_SECONDS=86400
+LOGIN_IP_BLOCK_RATE_LIMIT_RESET_INTERVAL_IN_SECONDS=86400
```

Comment:
We have quite a few parameters. For each rate limiter we have or we may add later, we have 3 env params
- rate limit - number of times an resource can be used
- pause  - pause time in seconds until the resource can be used again by the same user or IP
- reset interval - reset time in seconds, is the time until we reset the counter of resource usage by the same user/IP

Reply:
there are too many env variables related to rate limiting in my opinion and there will be even more based on this structure as we add it to more routes
I think we should have just a few: enabled rate limit, some global limitation and points and maybe some coeficient based on what the rate limit variables are calculated

what do you think? @adrianboros @lengyel-arpad85 @sanducb @beniaminmunteanu

Reply:
@dragosp1011  we have now only two optional env variables to enable rate limiting and three possible levels for limits